src/query/service/src/servers/http/v1/http_query_handlers.rs

```diff
@@ -267,7 +267,7 @@ async fn query_cancel_handler(
         let http_query_manager = HttpQueryManager::instance();
         match http_query_manager.get_query(&query_id).await {
             Some(query) => {
-                query.kill().await;
+                query.kill("http query cancel by handler").await;
                 http_query_manager.remove_query(&query_id).await;
                 StatusCode::OK
```

Comment:
What's the error will return to the client after killed? @youngsofun

Reply:
before this pr,  info will report 404 as since the query id can not be found .
to report timeout to client. we can retain the ids only for a longer time or with a limited count.